src/ContainerDeleteModal.jsx

```diff
@@ -18,7 +18,7 @@ const ContainerDeleteModal = ({ containerWillDelete, onAddNotification }) => {
         Dialogs.close();
         client.delContainer(container.isSystem, id, false)
                 .catch(ex => {
-                    const error = cockpit.format(_("Failed to remove container $0"), container.Names);
+                    const error = cockpit.format(_("Failed to remove container $0"), container.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)